syz-manager/manager.go

```diff
@@ -672,7 +672,10 @@ func (mgr *Manager) saveCrash(crash *Crash) bool {
 	}
 	if crash.Type == report.MultiRead {
 		mgr.mu.Lock()
-		mgr.multiReadFrames[crash.Frame] = true
+		mgr.multiReadFrames[crash.Frame] += 1
```

Comment:
What's the difference between multi-reads and memory leaks or data races? Why do we need to count them?

Reply:
from my understanding the difference that memory leaks and data races are ignored from the beginning we not even need to report any single crash for them, in case of Multi-read we want to make syzkaller report a crash for the first time this crash appears and then ignore it if it appears again